### DISCUSSION TITLE
Custom project file uploader: upload.jazzbutcher.com

### DISCUSSION THREAD

**Discussion by xpollen8** [xpollen8/jazzbutcher.com#55] [Announcements]
[upload.jazzbutcher.com](https://upload.jazzbutcher.com/) is an application I just created that:

- Allows you to login w/your github account
- Enter a github Issue \#
- Select a file to upload

This application will drop your upload directly into an AWS S3 bucket "inbox", and the files there will include the github number *and* your github account email. This will never be public, will be stripped from any final filename, and is used purely for proper Issue attribution.

If you're uploading FLAC audio, [See this note](https://github.com/xpollen8/jazzbutcher.com/discussions/62).

**Comment by wnerini** [comment]
I'll get the Frankfurt 85 show uploaded ASAP. 

_(the cover of  "Waiting For My Man" in that show is just totally tits.)_

**Comment by wnerini** [comment]
**QUESTION**
The first download is a folder-as-archive with 20 flac files a playlist, and 1 information text file. There are several more shows.

- Do we want to just upload the parts/show as we get them and mark the issue done when all the parts are there or should we wait until all the shows/parts are downloaded, and upload them all at once (potentially large zips file)?
- Do we want to upload each individual file, or create a zip archive for each show?

Thanks!

**Reply by xpollen8** [reply]
Ok. flac/mp4 added to allowed filetypes on upload

**Reply by xpollen8** [reply]
> **QUESTION** The first download is a folder-as-archive with 20 flac files a playlist, and 1 information text file. There are several more shows.
> 
> * Do we want to just upload the parts/show as we get them and mark the issue done when all the parts are there or should we wait until all the shows/parts are downloaded, and upload them all at once (potentially large zips file)?
> * Do we want to upload each individual file, or create a zip archive for each show?
> 
> Thanks!

After thinking about this... And because I'm uploading my first FLAC set right now.

Let's do this for FLAC bundles:

* Create an mp3 from the individual files `sox *.flac [DATE]_JazzButcher_[CITY]_FLAC.mp3`
* Upload the mp3
* Upload the original zipfile

This way we'll have the best of both worlds: something to listen to, AND the originals for someone who wants 'em.
